00-open-a-shapefile.Rmd

```diff
@@ -36,40 +37,36 @@ After completing this activity, you will:
 
```

Comment:
Grey box (all lessons): I updated some of the info if it was incorrect but I’m not adjusting formatting/ adding content sections to any lessons until the final format is decided on.